pages/content/amp-dev/documentation/components/index.html

```diff
@@ -19,6 +19,33 @@
     height: 1
     alt: lorem ipsum
   text: Each component reference page provides detailed information on how to integrate.
+
+
+highlight_teaser:
```

Comment:
The description for `amp-bodymovin-animation` is wrong, however the selected components should not be the highlighted ones in the first place 😉I'd vote for `amp-analytics`, `amp-autocomplete` and `amp-carousel` for the start.

Additionally, I only want to have a list of document links in `highlight_teaser` and pull all the teaser information from there instead of duplicating everything in the frontmatter here.